README.md

```diff
@@ -9,52 +7,50 @@
 
 ## Overview
 
-### Introduction
-
-*MITRE ATT&CK Threat Hunting Solution Pack* is designed to provide a set of threat hunting playbooks that demonstrates to users a variety of scenarios and use cases around threat hunting based on the information provided by the MITRE ATT&CK Framework.
+*MITRE ATT&CK Threat Hunting Solution Pack* is designed to provide a set of threat hunting playbooks that demonstrate a variety of scenarios and use cases around threat hunting based on the information provided by the MITRE ATT&CK Framework.
 
 Configure the data ingestion from SIEM using connectors such as Elasticsearch or Splunk. The hunting playbooks then trigger the hunt for the specified period mentioned in the hunt record and create an alert for the corresponding MITRE Technique.
 
-Refer to the following Simulation Scenarios' to experience the use case without any SIEM configuration.
+Refer to the following Simulation Scenarios to experience the use case without any SIEM configuration.
```

Comment:
This line and "This Solution Pack ships with the following simulation scenarios." are saying same thing again and again. Can we combine it?